index.html

```diff
@@ -6,7 +6,7 @@
     <meta name="description" content="A WebVR wave shooter game using A-Frame by Mozilla VR.">
     <meta name="theme-color" content="#3d4d55">
     <!--<script src="vendor/aframe-master.min.js"></script>-->
-    <script src="https://aframe.io/releases/0.7.0/aframe.min.js"></script>
+    <script src="https://rawgit.com/aframevr/aframe/3e7a4b3/dist/aframe-master.min.js"></script>
```

Comment:
The update to A-frame and THREE.js has some severe side-effects, including an audio problem `DOMException: Cannot decode detached ArrayBuffer` and graphical glitches. 

Attached is a screenshot of what it looks like for me. Strangely enough, once you go into VR, the graphical glitches disappear, but the high scores screen still is missing.
![image](https://user-images.githubusercontent.com/769957/39601677-967d4d98-4ed7-11e8-9631-deddaae07daa.png)